src/main/resources/schema.sql

```diff
@@ -1,8 +1,10 @@
-create table course
```

Comment:
Remove both schema.sql and data.sql files.

Reply:
done